### PR TITLE
create todo api

### PR DESCRIPTION
**Pull request by vindru**
## Description

_Create API for todo task_ POST /todos

## Database schema changes

_ todo: 
   {
      _id: string,
      title: string,
   }_

### Automated test cases added

- _POST /todos 
    input: {title: 'Adding first title'}
    output: success_
    
- _POST /todos 
    input: {title: ''}
    output: fail_

### Manual test cases run

- _NA_

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -11,9 +11,10 @@
...
     "PASSWORD_MISSING": "Please provide a password.",
-    "DUPLICATE_EMAIL": "The email you provided is already taken. Please provide a different one."
+    "DUPLICATE_EMAIL": "The email you provided is already taken. Please provide a different one.",
+    "INVALID_TITLE": "Please provide a title."
   },
   "MESSAGES": {
     "HEALTHCHECK": "OK"
   }
-}
+}
```
Comment (balwant-jtc) [inline src/locales/en.json]: Add new line

File: test/spec/todo/todo.spec.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      .request(expressApp)
+      .post('/todos')
+      .send({
+        title: '',
+      });
+
+    expect(res).to.have.status(400);
+    expect(res.body.message).to.equal('Please provide a title.');
+  });
+});
```
Comment (balwant-jtc) [inline test/spec/todo/todo.spec.ts:53]: Can add one test case for the valid string as well.

File: test/spec/todo/todo.spec.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  it('should return a validation error if tile is empty string.', async () => {
+    const res = await chai
+      .request(expressApp)
+      .post('/todos')
+      .send({
+        title: '',
+      });
+
+    expect(res).to.have.status(400);
+    expect(res.body.message).to.equal('Please provide a title.');
```
Comment (balwant-jtc) [inline test/spec/todo/todo.spec.ts]: I guess message will be inside failures array.

File: test/spec/todo/todo.spec.ts (hunk 2 above)
Comment (vindru) [reply test/spec/todo/todo.spec.ts:53]: added

**Pull request merged**